### DISCUSSION TITLE
Read Multi-rangerDeck and IMU data and transmit them to the ground station

### DISCUSSION THREAD

**Discussion by whutGuoao** [bitcraze/discussions#1693] [General]
Hello, the staff of the Crazyflie community. I want to get the Multi-ranger Deck and IMU data from Crazyflie's flight process for further processing. Is there any example code available for reference? As shown in the figure below, the image is sourced from https://store.bitcraze.io/products/stem-ranging-bundle
<img width="997" alt="微信图片_20241216113940" src="https://github.com/user-attachments/assets/a4d5fcf8-1a0e-4ede-ab63-fc4d62d2e741" />

**Comment by gemenerik** [comment]
I'm not sure about the plotter shown in the video, but you can definitely get logging data from the multi-ranger deck and IMU out using the logging framework. Check out [this step-by-step user guide](https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_connect_log_param/) for logging with the Python lib, or [set up logging in the Python client](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#logging) (and [write the logs to a file](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/logblocks_tab/))

**Reply by whutGuoao** [reply]
Okay, I understand. Thank you very much
